loadtest/control/gencontroller/controller.go

```diff
@@ -103,6 +103,15 @@ func (c *GenController) Run() {
 		c.createTeam,
 	}
 
+	cpaEnabled, resp := control.CustomProfileAttributesEnabled(c.user)
+	if resp.Err != nil {
+		c.sendFailStatus("Failed to retreive CustomProfileAttributesEnabled")
+		return
+	}
+	if cpaEnabled {
+		initActions = append(initActions, c.createCPAValues)
```

Comment:
Creates the CPA Values for a new users. I'm expecting this code to run once per user creation.

Reply:
`createCPAValues` creates various values.

In a real server this is either done manually by the user in UserPreferences. Or via LDAP/SAML sync. Or combination of both.